lib/commands/general.js

```diff
@@ -186,22 +186,25 @@ commands.setUrl = async function (url) {
   return await iosCommands.general.setUrl.call(this, url);
 };
 
-commands.getDevicePixelRatio = iosCommands.device.getDevicePixelRatio;
 commands.getViewportRect = iosCommands.device.getViewportRect;
 
 commands.getScreenHeight = async function () {
   return (await this.getWindowSize()).height;
 };
 
+commands.getScreenInfo = _.memoize(async function () {
```

Comment:
I wouldn't memoize it, since changing the display orientation might change the size of the status bar

Reply:
Also, the status bar is not always visible